src/backend/marsha/core/views.py

```diff
@@ -507,7 +507,7 @@ def _manage_exception(error):
 
         return {
             "app_data": json.dumps(app_data),
-            "static_base_url": f"{settings.STATIC_URL}js/",
+            "static_base_url": f"{settings.STATIC_URL}js/build/",
```

Comment:
Add test for the next this path will change 😉

Reply:
:+1: